cmd/cmd.go

```diff
@@ -797,7 +797,8 @@ func PullHandler(cmd *cobra.Command, args []string) error {
 
 			bar, ok := bars[resp.Digest]
 			if !ok {
-				bar = progress.NewBar(fmt.Sprintf("pulling %s...", resp.Digest[7:19]), resp.Total, resp.Completed)
+				digest := strings.TrimPrefix(resp.Digest, "sha256:")
+				bar = progress.NewBar(fmt.Sprintf("pulling %s...", digest), resp.Total, 0)
```

Comment:
Did we want to limit this to the first 8 bytes?

```suggestion
				bar = progress.NewBar(fmt.Sprintf("pulling %s...", digest), resp.Total, 0)
```

I see something like this with new client + old `ollama serve`

```
% time ./ollama pull llama2-uncensored
pulling manifest 
pulling 6aa74acf170f8fb8e6ff8dae9bc9ea918d3a14b6ba95d0b0287da31b09a4848c...  22% ▕████                 ▏ 825 MB/3.8 GB  103 MB/s     29
```

Reply:
I'll resolve this for the comment above since they are pointing out the same thing.